pkg/sse/v2alpha/sse_test.go

```diff
@@ -14,3 +14,79 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 package sse
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/eiffel-community/etos-api/internal/config"
+	"github.com/eiffel-community/etos-api/internal/stream"
+	"github.com/julienschmidt/httprouter"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+type cfg struct {
+	config.Config
+}
+
+func (c cfg) RabbitMQURI() string {
+	return ""
+}
+
+// TestSSECreateStream tests that the CreateStream endpoint works.
+// Does not test the authorization middleware!
```

Comment:
Not sure what the point of this comment is? If we going forward want to incorporate middleware testing in this test, then I'd rather the comment said this with a TODO included and/or a corresponding issue.

Reply:
It's just a warning so that we don't falsely believe that authorization is tested here.